tests/test_resolver.py

```diff
@@ -288,8 +288,8 @@ def test_template_conditions():
             "Bool": True,
             "BoolStr": "True",
             "IsEqualNum": {"Fn::Equals": [123456, 123456]},
-            "IsEqualStr": {"Fn::Equals": ["a", "a"]},
-            "IsEqualStr": {"Fn::Equals": [True, True]},
+            "IsEqualStr": {"Fn::Equals": ["a", "a"]},  # noqa: F601
+            "IsEqualStr": {"Fn::Equals": [True, True]},  # noqa: F601
```

Comment:
Does this actually work? Are both equals applied?

Reply:
Yep, but I just realized the bad naming.
the second one should be `"IsEqualBool": {"Fn::Equals": [True, True]},`